packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.ts

```diff
@@ -471,6 +471,11 @@ export default class TableColumnResizeEditing extends Plugin {
 		const editor = this.editor;
 		const modelTable = editor.editing.mapper.toModelElement( target.findAncestor( 'figure' )! )!;
 
+		// Do not resize if table model is in non-editable place.
+		if ( !editor.model.isSelectableEditable( modelTable ) ) {
+			return;
+		}
+
```

Comment:
How about image resize? It doesn't need any changes?

Reply:
It seems not needed.